src/components/policy/src/policy/usage_statistics/src/counter.cc

```diff
@@ -103,7 +103,7 @@ AppStopwatchImpl::AppStopwatchImpl(
 
 void AppStopwatchImpl::Start(AppStopwatchId stopwatch_type) {
   stopwatch_type_ = stopwatch_type;
-  timer_.Start(time_out_ * date_time::DateTime::MILLISECONDS_IN_SECOND, true);
+  timer_.Start(time_out_ * date_time::DateTime::MILLISECONDS_IN_SECOND, false);
```

Comment:
@malirod @pestOO 
What do you think about my idea to extend interface with 2 methods:
Start
StartRepeatable 
?